### ISSUE TITLE
unable to parse key-value

### ISSUE THREAD

**Issue by jetsh** [aler9/rtsp-simple-proxy#8]
Hi， I'm using uniview ip camera to test this program, but got the following output:
```
2020/06/22 14:47:08 rtsp-simple-proxy v0.3.7
2020/06/22 14:47:08 [UDP/RTP listener] opened on :8050
2020/06/22 14:47:08 [UDP/RTCP listener] opened on :8051
2020/06/22 14:47:08 [TCP listener] opened on :8554
2020/06/22 14:47:08 [STREAM test1] initializing with protocol tcp
2020/06/22 14:47:08 [STREAM test1] ERR: unable to setup authentication: unable to parse key-value (,nonce="133767111917411116111311118211673010032", stale="FALSE")
2020/06/22 14:47:13 [STREAM test1] initializing with protocol tcp
2020/06/22 14:47:13 [STREAM test1] ERR: unable to setup authentication: unable to parse key-value (,nonce="1337961385111862179317211191311699045743", stale="FALSE")
2020/06/22 14:47:18 [STREAM test1] initializing with protocol tcp
2020/06/22 14:47:18 [STREAM test1] ERR: unable to setup authentication: unable to parse key-value (,nonce="133771274111532192614111111111883456838", stale="FALSE")
```
My conf is here:
```
# timeout of read operations
readTimeout: 5s
# timeout of write operations
writeTimeout: 5s

server:
  # supported protocols
  protocols: [ tcp, udp ]
  # port of the RTSP TCP listener
  rtspPort: 8554
  # port of the RTP UDP listener
  rtpPort: 8050
  # port of the RTCP UDP listener
  rtcpPort: 8051
  # optional username required to read
  readUser:
  # optional password required to read
  readPass:

streams:
  # name of the stream
  test1:
    # url of the source stream, in the format rtsp://user:pass@host:port/path
    url: 'rtsp://xxx:xxx@30.207.67.154'
    # whether to use tcp or udp
    protocol: tcp
```
The RTSP url works fine in VLC. Any help will be  appreciated.

**Comment by aler9** [comment]
Hi, this should be fixed in v0.3.9. Please try again and let me know. Thanks.

**Comment by jetsh** [comment]
It's working! Thanks for your help.
